ext/dom/php_dom.stub.php

```diff
@@ -485,7 +485,7 @@ class DOMCharacterData extends DOMNode implements DOMChildNode
     public ?DOMElement $nextElementSibling;
 
     /** @tentative-return-type */
-    public function appendData(string $data): bool {}
+    public function appendData(string $data): true {}
```

Comment:
@Girgias Do you have an idea when and how we could migrate these return types to `void`? Does it make sense to first migrate them to tentative `true`?

Reply:
Probably only PHP 9.0. But marking them as ``true`` indicates that the return value is pointless.

The issue with going from ``true`` to ``void`` is that if the return value is used it goes from ``true`` to ``null`` which if checked in a condition will now hit the other branch :/

Reply:
Yeah, I see the problem :) I was mainly interested in whether you see the benefit of the slight BC break of changing non-final method return types to true before finally marking them as void.

Once, I came up with an idea to check whether the return type is used (`RETURN_VALUE_USED`): if so, then we could add a deprecation notice. I think this is the best possibility we have, but Nikita wasn't exactly happy with it back then because it doesn't work in all cases.